test/instabug_test.dart

```diff
@@ -257,13 +259,48 @@ void main() {
   });
 
   test('[clearAllExperiments] should call host method', () async {
+    // ignore: deprecated_member_use_from_same_package
     await Instabug.clearAllExperiments();
 
     verify(
       mHost.clearAllExperiments(),
     ).called(1);
   });
 
+  test('[addFeatureFlags] should call host method', () async {
+    await Instabug.addFeatureFlags(
+      List.of([
+        FeatureFlag(name: 'name1', variant: 'variant1'),
+        FeatureFlag(name: 'name2', variant: 'variant2'),
+      ]),
+    );
+
```

Comment:
Can we review the featureFlags content to confirm the addition?

Reply:
There is no way to export feature tags from the native SDK to verify whether feature tags have been removed or added